target/linux/realtek/files-5.15/arch/mips/rtl838x/prom.c

```diff
@@ -186,7 +186,6 @@ void __init prom_init(void)
 
 	/* Early detection of CMP support */
 	if(soc_info.family == RTL9310_FAMILY_ID) {
-		mips_cm_probe();
```

Comment:
generic improvement? I think we can merge this patch anyway. How is this on v5.10? Does this also apply to v5.10?